radar-auth/src/main/java/org/radarbase/auth/jwks/JwksTokenVerifierLoader.kt

```diff
@@ -78,7 +78,7 @@ class JwksTokenVerifierLoader(
 
         private val logger = LoggerFactory.getLogger(JwksTokenVerifierLoader::class.java)
 
-        private val httpClient = HttpClient(CIO).config {
+        private fun httpClient(url: String) = HttpClient(CIO).config {
```

Comment:
you can move this to the class property instead of converting to a function. I don't think the url will change per class instance